crates/wat/src/lib.rs

```diff
@@ -235,7 +235,12 @@ impl Parser {
             }),
         })?;
         match self.parse_bytes(Some(file), &contents) {
-            Ok(bytes) => Ok(bytes.into_owned()),
+            Ok(Cow::Borrowed(bytes)) => {
```

Comment:
```suggestion
            // removes a copy performed when the input file is WebAssembly when parsing input files on the CLI. This happened due to the implementation of Cow::to_owned but isn't necessary since we have the original owned copy around to return that instead.
            Ok(Cow::Borrowed(bytes)) => {
```